deployment/aws-waf-security-automations-firehose-athena.template

```diff
@@ -419,7 +419,7 @@ Resources:
       Type: AWS::Athena::WorkGroup
       Condition: HttpFloodAthenaLogParser
       Properties:
-        Name: WAFLogAthenaQueryWorkGroup
+        Name: !Join ['-', ['WAFLogAthenaQueryWorkGroup', !Ref UUID]]
```

Comment:
Just to add here, that these groups are exported and fed in to the acl stack. So these values are not getting hard-coded so I don't see why we couldn't make them unique.

Reply:
@aijunpeng I see you have previously commented on #150 about this change. Any chance we can reconsider. The setup is not hardcoded to those names so no reason we can't make the update, give they are already exported from the stack to use else where. 

Understand you don't want to support multiple instances in one region. It could though be useful to help with testing updates to WAF config in a region without having to go through the pain of setting up everything in a new account or region.